now.json

```diff
@@ -0,0 +1,12 @@
+{
+    "name": "flamingo",
+    "version": 2,
+    "builds": [
+      {
+        "src": "build.sh",
```

Comment:
Should we wrap this into a `script/` folder?
We might have more of this latter, wdyt?

Reply:
Agreed. Done. ✅